### ISSUE TITLE
efficientdet-d0 training print nan after few steps

### ISSUE THREAD

**Issue by sangyo1** [EdjeElectronics/TensorFlow-Lite-Object-Detection-on-Android-and-Raspberry-Pi#183]
So I am trying to use the efficientdet-d0 with own label dataset, when I train it it is working fine on first few thousands steps, but it print nan all the losses after training. I am curious of the steps size, if I decrease the stepsizes it occurs after 3000 steps, but if I increase steps upto 30,000 then it occurs after 6000 steps. I am not sure how to fixing it, or what should I have to do?

`I0409 23:03:12.869635 140410452137792 model_lib_v2.py:708] {'Loss/classification_loss': nan,
 'Loss/localization_loss': nan,
 'Loss/regularization_loss': nan,
 'Loss/total_loss': nan,
 'learning_rate': 0.07922263}
`

Do I have to change something in this line of code from `def rewrite_config(num_steps, batch_size, num_classes, train_record_fname, val_record_fname)` to fix that problem?
```
        if chosen_model == 'efficientdet-d0':
            s = re.sub('keep_aspect_ratio_resizer', 'fixed_shape_resizer', s)
            s = re.sub('pad_to_max_dimension: true', '', s)
            s = re.sub('min_dimension', 'height', s)
            s = re.sub('max_dimension', 'width', s)

        f.write(s)
```

**Comment by sangyo1** [comment]
increased the batch helped
